docs/tutorials/dimension-templates.md

```diff
@@ -0,0 +1,168 @@
+# Use dimension templates to create dynamic alarms
+
+Your ability to monitor the health of your systems and applications relies on your ability to create and maintain
+the best set of alarms for your particular needs.
+
+In v1.18 of Netdata, we introduced **dimension templates** for alarms, which simplifies the process of writing [alarm
+entities](../../health/README.md#entities-in-the-health-files) for charts with many dimensions.
+
+Dimension templates can condense many individual entities into one—no more copy-pasting one entity and changing the
+`alarm`/`template` and `lookup` lines for each dimension you'd like to monitor.
+
+They are, however, an advanced health monitoring feature. For more basic instructions on creating your first alarm,
+check out our [health monitoring documentation](../../health/), which also includes
+[examples](../../health/README.md#examples).
+
+> For more information about dimension templates, including examples of before/after examples, see our [v1.18
+> announcement post](https://blog.netdata.cloud/posts/release-1.18/).
+
+## The fundamentals of `foreach`
+
+Our dimension templates update creates a new `foreach` parameter to the existing [`lookup`
+line](../../health/README.md#alarm-line-lookup). This is where the magic happens.
+
+You use the `foreach` parameter to specify which dimensions you want to monitor with this single alarm. You can separate
+them with a comma (`,`) or a pipe (`|`), or you can use a wildcard (`*`) to say that you want an alarm for every
```

Comment:
Instead to say that the person can use "wildcard", I would suggest we say that the user can use "Netdata simple pattern" and link to our documentation.